samples/crypto/psa_tls/README.rst

```diff
@@ -29,6 +29,217 @@ Overview
 The sample can act as either a network server or a network client.
 By default, the sample is configured to act as a server.
 
+TLS version support
+===================
+
+The sample supports both TLS v1.2 and TLS v1.3.
+See the following table for the support overview for each compatible board target.
+The table mirrors the test setup used for TLS verification.
+
+.. tabs::
+
+   .. tab:: TLS v1.2
+
+      .. list-table::
+         :header-rows: 1
+         :widths: auto
+
+         * - Board target
+           - Backend
+           - CMSE supported
+           - DTLS supported
+           - Limitations / not supported
+           - Comments
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`Mbed TLS legacy c33xx backend <nrf_security_backends_cc3xx>`
+           - No
+           - No
+           - AES256, AES-GCM, SHA-512
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`Mbed TLS legacy oberon backend<nrf_security_backends_oberon>`
+           - No
+           - No
+           -
+           -
+         * - ``nrf52840dk/nrf52840``
+           - :ref:`PSA Crypto cc3xx driver<nrf_security_drivers_cc3xx>`
+           - No (secure only)
+           - Yes
+           - RSA
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`PSA Crypto cc3xx driver<nrf_security_drivers_cc3xx>`
+           - Yes
+           - Yes
+           - RSA
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf5340dk/nrf5340/cpuapp``
+           - :ref:`Mbed TLS legacy c33xx backend <nrf_security_backends_cc3xx>`
+           - No
+           - No
+           -
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
```

Comment:
Leave this comment blank. Same reasoning as above (this row lists legacy crypto support. Not PSA crypto)